src/components/Expensicons/Icons.js

```diff
@@ -0,0 +1,39 @@
+import ICON_NAMES from './ICON_NAMES';
```

Comment:
strings constants usually go into `CONST`. I wouldn't be opposed to breaking up the constants file at some point to organize them better. But we have a well established practice of putting all string constants into a single file and should maybe keep rolling with that for now.